diff --git a/src/camel/interpreter/interpreter.py b/src/camel/interpreter/interpreter.py
index 73d88e1..3354386 100644
--- a/src/camel/interpreter/interpreter.py
+++ b/src/camel/interpreter/interpreter.py
@@ -340,7 +340,7 @@ def _eval_constant(
             return EvalResult(
                 result.Error(
                     CaMeLException(
-                        NotImplementedError(f"unsupported constant type {type(node.value).__type__}"),
+                        NotImplementedError(f"unsupported constant type {type(node.value).__name__}"),
                         (node,),
                         (),
                     )
